packages/components/src/components/hds/alert/index.ts

```diff
@@ -156,11 +137,7 @@ export default class HdsAlert extends Component<HdsAlertSignature> {
     }
   }
 
-  /**
-   * Get the class names to apply to the component.
-   * @method Alert#classNames
-   * @return {string} The "class" attribute to apply to the component.
-   */
+  // The "class" attribute to apply to the component.
```

Comment:
[suggestion/non-blocker] I would leave these comments only for snippets of code for which it may not be clear at first sight what they do; in this case, it's a pretty common/widespread function.